steamsync/steamsync/launchers/itch.py

```diff
@@ -52,6 +51,9 @@ def collect_games(self) -> list[defs.GameDefinition]:
                 if not exes:
                     print(f"Warning: Failed to find executable for game '{title}'.")
                     continue
+                if len(exes) > 1:
+                    # Ignore Godot's extra game.console.exe executable.
+                    exes = [exe for exe in exes if exe.name.endswith(".console.exe")]
```

Comment:
Doesn't this do the opposite? 

```suggestion
                    exes = [exe for exe in exes if not exe.name.endswith(".console.exe")]
```

Reply:
Good catch! Thanks!